assets/js/wc-gateway-ppec-checkout-handler.js

```diff
@@ -0,0 +1,8 @@
+jQuery(document).ready(function($) {
+    $( '#order_review' ).on('change', 'input[name=payment_method]', function() {
+    	if ( $('#payment_method_ppec_paypal').size() ) {
+    		$('body').trigger( 'update_checkout' );
+			// TODO: Update billing fields
```

Comment:
@claudiulodro any ideas how I can achieve this?

Reply:
Are you trying to refresh the billing fields? Should be able to do that with something like this:

```
jQuery( document.body ).trigger(  
    'country_to_state_changing', 
    [ jQuery( '#billing_country' ).val(), jQuery( '.woocommerce-billing-fields' ) ] 
)
```

The `country_to_state_changing` action handles updating the form when changing countries. Triggering it with the current country should reset them. If that doesn't meet your needs you'll likely have to go with something custom, as WC probably doesn't have what you're looking for built in.